Source/Controllers/MainViewControllers/TableContent/SPTableContent.m

```diff
@@ -745,6 +745,24 @@ - (void) loadTableValues
 		[queryString appendFormat:@" ORDER BY %@", [[[dataColumns safeObjectAtIndex:[sortCol integerValue]] safeObjectForKey:@"name"] backtickQuotedString]];
 		if (isDesc) [queryString appendString:@" DESC"];
 	}
+    else // no sorting is set
+    {
+        if ([prefs boolForKey:@"SortDescendingByPrimaryKey"])
```

Comment:
For preferences, we shouldn't refer to the values by string names - please add the new item to the SPConstants file (following the existing style convention) and leverage the constant when referring to the pref